apache2/re_operators.c

```diff
@@ -4231,11 +4231,6 @@ static int msre_op_fuzzy_hash_init(msre_rule *rule, char **error_msg)
     return 1;
 #endif
     return 1;
-
-invalid_parameters:
```

Comment:
It seems like this label is still referenced [here](https://github.com/owasp-modsecurity/ModSecurity/blob/82c5c3a6dec1c68c081dc69eee5faf9a34818852/apache2/re_operators.c#L4171) and [here](https://github.com/owasp-modsecurity/ModSecurity/blob/82c5c3a6dec1c68c081dc69eee5faf9a34818852/apache2/re_operators.c#L4188). Do we really want to remove this?

Reply:
Ah, I missed that because I don't have that feature enabled. I'll restore this.